common/error.cpp

```diff
@@ -186,3 +188,5 @@ namespace pdo {
         } // ThrowSgxError
     }
 } // namespace pdo
+
+#endif // _CLIENT_ONLY_
```

Comment:
Shouldn't we just omit `error.cpp` from the client build in cmake? I'd imagine stdlib and stdio are imported into the client through other files.

Reply:
More concerned about how/where this gets included in other common source.